connection.go

```diff
@@ -282,14 +287,18 @@ func (c *Connection) handleClose(fd int) {
 	if c.connected.Get() {
 		c.connected.Set(false)
 		c.loop.DeleteFdInLoop(fd)
-
 		c.callBack.OnClose(c)
 		if err := unix.Close(fd); err != nil {
 			log.Error("[close fd]", err)
 		}
-
 		ringbuffer.PutInPool(c.inBuffer)
 		ringbuffer.PutInPool(c.outBuffer)
+		if c.idleTime > 0 {
```

Comment:
Maybe we don't need it now, because we just need to check `c.timer`

Reply:
done, have simplified the code